src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -484,6 +484,10 @@ private string QuoteName(object name)
         {
             if (name == null)
                 return "''";
+            else if (name is Uri)
```

Comment:
In this case, `name` may or may not be a Uri.  I'm special casing when it is a Uri, I want to use the AbsoluteUri member value.